src/main/scala/nl.knaw.dans.easy.solr4files/EasySolr4filesIndexApp.scala

```diff
@@ -113,7 +113,9 @@ trait EasySolr4filesIndexApp extends ApplicationWiring with AutoCloseable
       .flatMap(getFileItem(bag, _).toSeq) // skip the None's and unwrap the Some's
       .map(_.flatMap(createDoc(_, ddm))) // createDoc if getFileItem did not fail
       .takeUntilFailure
-      .doIfFailure { case MixedResultsException(results: Seq[_], _) =>
+      .doIfFailure { case MixedResultsException(results: Seq[_], e: Throwable) =>
+        logger.error(e.getMessage)
+        logger.info("Indexing will be stopped. The files up till this error have been submitted; the list of submitted files here below:")
```

Comment:
```suggestion
        logger.info("One of the files could not be indexed. Only the following files are submitted:")
```

Reply:
```suggestion
        logger.info("Indexing has stopped, because one of the files could not be indexed. Only the following files are submitted:")
```